okapi-core/src/main/java/org/folio/okapi/managers/TenantManager.java

```diff
@@ -385,6 +385,8 @@ private void waitTenantInit(Tenant tenant, ModuleInstance getInstance,
     if (!options.checkInvoke(md.getId())) {
       return Future.succeededFuture();
     }
+    long startTime = System.nanoTime();
```

Comment:
System.currentTimeMillis() is good enough: https://stackoverflow.com/questions/351565